### DISCUSSION TITLE
Installing Odin

### DISCUSSION THREAD

**Discussion by atmersonatienza** [mbround18/valheim-docker#653] [Q&A] [answered]
Hey everyone,

I'm currently following the instructions to install Odin via https://github.com/mbround18/valheim-docker/blob/main/src/odin/README.md. However, I'm running into an issue when running `makers -e production release`. It's throwing a `no makefile found (tried GNUmakefile, makefile, Makefile` error. I'm also running the makers command inside the repo. Would it be better to install via bundle.zip? 

Thanks!

**Comment by mbround18** [comment, marked as answer]
yes very easy to install via the bundle.zip, the makers stuff you need:
- Rust
- cargo-make crate
- and a few libs documented in the readme.

if you have makers command already present try cargo make -e production release
